front/src/common/InfraSelector/InfraSelector.tsx

```diff
@@ -1,38 +1,39 @@
 import React, { useState, useEffect } from 'react';
-import { PropTypes } from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 import { setFailure } from 'reducers/main';
-import { get } from 'common/requests';
 import icon from 'assets/pictures/components/tracks.svg';
 import InfraSelectorModal from 'common/InfraSelector/InfraSelectorModal';
 import { useModal } from 'common/BootstrapSNCF/ModalSNCF';
 import { getInfraID } from 'reducers/osrdconf/selectors';
 import { FaLock } from 'react-icons/fa';
 import './InfraSelector.scss';
-import { INFRA_URL } from './Consts';
+import { Infra, osrdEditoastApi } from 'common/api/osrdEditoastApi';
 
-export default function InfraSelector(props) {
-  const { modalOnly } = props;
+type InfraSelectorProps = {
+  modalOnly?: boolean;
+};
+
+const InfraSelector = ({ modalOnly }: InfraSelectorProps) => {
```

Comment:
I think you miss to reimplement the modalOnly default value to false

Reply:
Done !!!